Scripts/XRTrailRenderer.cs

```diff
@@ -20,131 +22,84 @@ public class VRTrailRenderer : MonoBehaviour
     bool m_StealLastPointWhenEmpty = true;
 
     [SerializeField]
-    Color[] m_Colors;
+    [Tooltip("How long the tail should be (second) [ 0, infinity ].")]
+    float m_Time = 5.0f;
+
+    [SerializeField]
+    [Tooltip("The minimum distance to spawn a new point on the trail [ 0, infinity ].")]
+    float m_MinVertexDistance = 0.1f;
+
+    [SerializeField]
+    [Tooltip("Destroy GameObject when there is no trail?")]
+    bool m_Autodestruct = false;
```

Comment:
this is a field on the traditional TrailRenderer, but it doesn't seem to actually do anything here -  i see no references to it that would handle destroying it, and the GameObject certainly remains even when there is no trail